content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -551,6 +551,12 @@ The reset process does not reset or clean up iptables rules or IPVS tables. If y
 iptables -F && iptables -t nat -F && iptables -t mangle -F && iptables -X
 ```
 
+If you wish to reset IPVS tables,you must do so manually to reset your system's IPVS tables:
```

Comment:
please change to:
```
If you wish to reset IPVS tables, you must call the following command:
```

Reply:
Suggested nit: To reset IPVS tables, call the following command:

Reply:
@neolit123 @Rajakavitha1  Thank you so much.
```
If you wish to reset IPVS tables, you must call the following command:
```

Reply:
@neolit123 I have corrected this description!